package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nethsecurity-ui",
-  "version": "0.0.0",
+  "version": "1.0.27",
```

Comment:
The latest tag version is 1.0.28

Reply:
was a leftover test just to check if release please will follow the schema :cry: